### PR TITLE
Create Card.tsx

### PR DESCRIPTION
**Pull request by efrenmercadojr**
## TODO
1. Implement card component similar to https://setravelapp.herokuapp.com/browse
2. Use tailwind css for styling

Tailwind Crash Course: https://www.youtube.com/watch?v=UBOj6rqRUME
Tailwind Docs: https://tailwindcss.com/docs/

## Screenshots

Add screenshots here

### CODE REVIEW

**Comment by efrenmercadojr** [comment]
@FutureHat Add screenshots. Click "Ready for review" if tapos na.

File: src/components/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg m-1">
+      <img
+        src="https://gttp.imgix.net/225058/x/0/bohol-balicasag-balicasag-island-natanielluperte-2020-wm-1.jpg"
+        alt=""
+        className="w-full"
+      />
+      <div className="px-6 py-6 center">
+        <p className="tour-title text-lg">
+          <strong> Baler, Aurora Tour Package</strong>
```
Comment (efrenmercadojr) [inline src/components/Card.tsx:25]:
```suggestion
          <strong>{title}</strong>
```

File: src/components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <p className="text-justify truncate ">{description}</p>
+        <div className="py-5 flex w-full ">
+          <div className="w-1/2">
+            <button className="bg-blue-600 hover:bg-blue-700  rounded text-white px-2 py-1 ">
+              Read More
+            </button>
+          </div>
+          <div className="w-3/5 "></div>
+          <div className="">
+            <p className="font-bold text-xl "> P1599</p>
```
Comment (efrenmercadojr) [inline src/components/Card.tsx:39]:
```suggestion
            <p className="font-bold text-xl ">{price}</p>
```

File: src/components/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      />
+      <div className="px-6 py-6 center">
+        <p className="tour-title text-lg">
+          <strong> Baler, Aurora Tour Package</strong>
+        </p>
+        <p className="tour-organization text-md text-gray-600">
+          {organization}
+        </p>
+        <p className="text-justify truncate ">{description}</p>
+        <div className="py-5 flex w-full ">
```
Comment (efrenmercadojr) [inline src/components/Card.tsx:31]:
```suggestion
        <div className="py-5 flex w-full">
```
@FutureHat  Para san yung mga extra space? :thinking:

**Review by FutureHat — APPROVED** [review]

**Review by FutureHat — APPROVED** [review]

File: src/components/Card.tsx (hunk 3 above)
Comment (FutureHat) [reply src/components/Card.tsx:31]: @efrenmercadojr may binura ata akong code dyan boss and hindi nasama space

**Review by FutureHat — APPROVED** [review]

**Review by FutureHat — APPROVED** [review]

**Review by FutureHat — APPROVED** [review]

**Comment by efrenmercadojr** [comment]
@FutureHat Di ikaw mag aapprove. Haha

File: src/components/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  image: string;
+  organization: string;
+  description: string;
+  price: string;
+}>): ReactElement => {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg m-1">
+      <img src={image} alt="" className="w-full" />
+      <div className="px-6 py-6 center">
+        <p className="tour-title text-lg">
```
Comment (efrenmercadojr) [inline src/components/Card.tsx:20]:
```suggestion
        <p className="text-lg">
```

@FutureHat Wag ka naglalagay ng class na wala namang pag gagamitan

**Comment by efrenmercadojr** [comment]
@FutureHat Screenshots

**Pull request merged**